config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -115,13 +116,6 @@ case ${BOARD} in
     ;;
 esac
 
-# if environment variable SANDBOX_BUG is set, then we need to use sandbox workaround
```

Comment:
Isn't it still needed? In my experience the R116 fail without the sandbox workaround.

Reply:
As i know this sandbox issue and following ones was caused by lack of vm/max_map_count tuning. I got trogdor built without any sandbox workarounds after adjusting this value.
I updated my tool to adjust this value as well (as SDK cannot do it inside docker image).
Can you test that on your side too? (using latest version of kci-cros-builder.py tool)

Reply:
Ah right, I tested without the workaround and builds run fine now. So yes, this can go away.